### PR TITLE
[Gallery] Rebrand and enhancements

### PR DESCRIPTION
**Pull request by niels9001**
- New home page design
- Improvements in markdown styling
- New sample page design

### CODE REVIEW

File: CommunityToolkit.App.Shared/Pages/TabbedPage.xaml [hunk 1]
```diff
@@ -54,13 +54,9 @@
...
-        <controls:TitleBar x:Name="appTitleBar"
-                           Title="Toolkit Labs Gallery">
-            <controls:TitleBar.Icon>
-                <BitmapIcon ShowAsMonochrome="False"
-                            UriSource="ms-appx:///Assets/AppTitleBar.png" />
-            </controls:TitleBar.Icon>
-        </controls:TitleBar>
+        <controls:TitleBar Title="Windows Community Toolkit Gallery"
+                           VerticalAlignment="Top"
+                           Icon="ms-appx:///Assets/AppTitleBar.png" />
```
Comment (michael-hawker) [inline CommunityToolkit.App.Shared/Pages/TabbedPage.xaml:59]:
```suggestion
        <controls:TitleBar x:Name="appTitleBar"
                           Title="Windows Community Toolkit Gallery"
                           VerticalAlignment="Top"
                           Icon="ms-appx:///Assets/AppTitleBar.png" />
```

**Comment by michael-hawker** [comment]
Adding TitleBar name fixed the first issue, but now both CIs show the same error around Uno flipping out with something on the TabbedPage?

**Comment by michael-hawker** [comment]
Rebased on latest main

File: ProjectHeads/App.Head.props [hunk 2]
```diff
@@ -40,7 +40,18 @@
       </ItemGroup>
     </When>
   </Choose>
-
+  <Choose>
+    <When Condition="'$(ToolkitSettingsControlsSourceProject)' == ''">
+      <ItemGroup>
+        <PackageReference Include="CommunityToolkit.$(DependencyVariant).Controls.SettingsControls" Version="8.0.0-beta.1"/>
```
Comment (michael-hawker) [inline ProjectHeads/App.Head.props:46]: There's two possible problems here:
- Version mismatch 8.0.230801-preview
- Transitive dependencies for the Settings Controls

**Comment by michael-hawker** [comment]
Opened https://github.com/CommunityToolkit/Windows/pull/189 to test these on the main repo before merging.

**Comment by niels9001** [comment]
> Opened https://github.com/CommunityToolkit/Windows/pull/189 to test these on the main repo before merging.
> 
> @niels9001 is there other properties or changes that need to be made to the config in the main repo?

Should be good.. have tested it locally with various branches and that seemed to work fine.

**Comment by michael-hawker** [comment]
Linux build is failing on the main repo: https://github.com/CommunityToolkit/Windows/actions/runs/5968507459/job/16192469795?pr=189 - probably related to the double-include, check the top pattern and ensure to include the extra conditionals?

File: ProjectHeads/App.Head.props [hunk 3]
```diff
@@ -40,7 +40,42 @@
...
     </When>
   </Choose>
-
+  <Choose>
+    <When Condition="'$(ToolkitSettingsControlsSourceProject)' == ''">
+      <ItemGroup>
+        <PackageReference Include="CommunityToolkit.$(DependencyVariant).Controls.SettingsControls" Version="8.0.230823-rc"/>
+      </ItemGroup>
+    </When>
+    <Otherwise>
```
Comment (michael-hawker) [inline ProjectHeads/App.Head.props:49]: These all need to follow the pattern above, like the Converters:

`<When Condition="'$(IsSingleExperimentHead)' == 'true' and $(MSBuildProjectName.StartsWith('Converters')) == 'false'">`

**Review by michael-hawker — CHANGES_REQUESTED** [review]
See comment above

File: ProjectHeads/App.Head.props (hunk 3 above)
Comment (niels9001) [reply ProjectHeads/App.Head.props]: Oh whoops, added that

File: ProjectHeads/App.Head.props [hunk 4]
```diff
@@ -40,7 +40,42 @@
...
+      </ItemGroup>
+    </When>
+  </Choose>
+   <Choose>
+    <When Condition="'$(ToolkitTriggersSourceProject)' == ''">
+      <ItemGroup>
+        <PackageReference Include="CommunityToolkit.$(DependencyVariant).Triggers" Version="8.0.230823-rc"/>
+      </ItemGroup>
+    </When>
+    <When Condition="'$(IsSingleExperimentHead)' == 'true' and $(MSBuildProjectName.StartsWith('ToolkitTriggersSourceProject')) == 'false'">
```
Comment (michael-hawker) [inline ProjectHeads/App.Head.props:73]:
```suggestion
    <When Condition="'$(IsSingleExperimentHead)' == 'true' and $(MSBuildProjectName.StartsWith('Triggers')) == 'false'">
```
Fix the condition here.

**Comment by michael-hawker** [comment]
https://github.com/CommunityToolkit/Windows/pull/189 passed, so changes should be good. I'll take things for a spin here in a bit and get this merged in.

**Review by michael-hawker — APPROVED** [review]
Looks good! I've been noting some general things we'll want to fix, but will open new issues to work forward.

**Pull request merged**